core/src/main/scala/com/nvidia/spark/rapids/tool/qualification/QualOutputWriter.scala

```diff
@@ -372,6 +375,7 @@ object QualOutputWriter {
   val GPU_OPPORTUNITY_STR_SIZE: Int = GPU_OPPORTUNITY_STR.size
   val UNSUPPORTED_EXECS_MAX_SIZE: Int = 25
   val UNSUPPORTED_EXPRS_MAX_SIZE: Int = 25
+  val ESTIMATED_FREQUENCY_MAX_SIZE: Int = 33
```

Comment:
I kept thinking a month should be 30. why 33 :D 

Suggest this to be dynamically calculated

```
val ESTIMATED_FREQUENCY_MAX_SIZE: Int = ESTIMATED_FREQUENCY.size
```